Baikal/PostEffects/ML/denoiser.cpp

```diff
@@ -403,5 +423,36 @@ namespace Baikal
         {
 
         }
+
+        void MLDenoiser::SetParameter(std::string const& name, Param value)
+        {
+            auto param = GetParameter(name);
+
+            switch (value.GetType())
```

Comment:
No need to check parameters, just set the dirty flag. The parameters are changed rarely.

Reply:
Done